object-filter/src/main/java/org/infinispan/objectfilter/impl/aggregation/AvgAccumulator.java

```diff
@@ -0,0 +1,51 @@
+package org.infinispan.objectfilter.impl.aggregation;
+
+/**
+ * @author anistor@redhat.com
+ * @since 8.0
+ */
+public final class AvgAccumulator extends FieldAccumulator {
+
+   private static class Avg {
```

Comment:
I have not fixed it here yet, but I added a jira: https://issues.jboss.org/browse/ISPN-5629